modules/apps/image/image-impl/src/main/java/com/liferay/image/internal/ImageMagickImpl.java

```diff
@@ -226,7 +227,16 @@ public byte[] scale(byte[] bytes, String mimeType, int width, int height)
 
 			Future<?> future = convert(arguments);
 
-			future.get();
+			if (_log.isDebugEnabled()) {
+				ProcessEvent processEvent = (ProcessEvent)future.get();
+
+				if (processEvent.getException() != null) {
+					_log.debug(processEvent.getException());
+				}
+			}
+			else {
+				future.get();
+			}
```

Comment:
Log blocks should not contain logic not related to logging. Can you refactor to something like this?
```
ProcessEvent processEvent = (ProcessEvent)future.get();

if (_log.isDebugEnabled() && (processEvent.getException() != ...)) {
     _log.debug(...);
}
```
This way, it is more clear that we can remove the debug statements without affecting the behavior.